src/backend/data_preprocessing.py

```diff
@@ -1,8 +1,8 @@
-from backend.database_connection import DatabaseConnection
+from database_connection import DatabaseConnection
 import os
-from backend.histogram_of_gradients import HistogramOfGradients
+from histogram_of_gradients import HistogramOfGradients
 from multiprocessing import Process
-from backend.utils import get_image_directory
+from utils import get_image_directory
```

Comment:
from . import utils

and then u can use utils.get_image_directory() function

If we use this then we don't need to import all the required functions every time.

Reply:
Again relative path doesnt work as we havent defined the package